src/components/ContentNode/LinkableHeading.vue

```diff
@@ -9,24 +9,25 @@
 -->
 
 <template>
-  <component
-    :id="anchor"
-    :is="`h${level}`"
-  >
+  <div class="heading-wrapper">
+    <component
+      :id="anchor"
+      :is="`h${level}`"
+      class="heading"
+    >
+      <slot />
+    </component>
     <router-link
       v-if="shouldLink"
       :to="{ hash: `#${anchor}` }"
-      class="header-anchor"
-      aria-label="Scroll to section"
+      class="anchor"
       @click="handleFocusAndScroll(anchor)"
     >
-      <slot />
       <LinkIcon class="icon" aria-hidden="true" />
+      <span :aria-labelledby="anchor" class="visuallyhidden" />
```

Comment:
cant we just put an aria-label on the link?

Reply:
I decided to keep the previous style fixing the AX issue https://github.com/apple/swift-docc-render/pull/698/commits/f9ce461bd83c90d5331108648b90b58d171abcf3